modules/core/src/main/scala/lucuma/odb/api/model/ParallaxModel.scala

```diff
@@ -3,9 +3,10 @@
 
 package lucuma.odb.api.model
 
+import cats.Eq
 import lucuma.core.math.{Angle, Parallax}
 import lucuma.core.util.{Display, Enumerated}
-import cats.syntax.validated._
+import cats.implicits._
```

Comment:
I was under the impression we were moving to `import cats.syntax.all._`.

Reply:
Thanks for spotting that.  It was a wrong turn so I've put it back the way it was.